src/components/Wallet/ExtendedPublicKeySelector.jsx

```diff
@@ -24,24 +24,26 @@ import SignatureImporter from "../ScriptExplorer/SignatureImporter";
 class ExtendedPublicKeySelector extends React.Component {
   static propTypes = {
     extendedPublicKeyImporters: PropTypes.shape({}).isRequired,
-    inputs: PropTypes.arrayOf({
-      bip32Path: PropTypes.string,
-    }).isRequired,
-    onChange: PropTypes.func.isRequired,
+    inputs: PropTypes.arrayOf(
+      PropTypes.shape({
+        bip32Path: PropTypes.string,
+      })
+    ).isRequired,
+    onChange: PropTypes.func,
     network: PropTypes.string.isRequired,
     number: PropTypes.number.isRequired,
     setBIP32Path: PropTypes.func.isRequired,
     setMethod: PropTypes.func.isRequired,
     setSigningKey: PropTypes.func.isRequired,
-    signatureImporters: PropTypes.arrayOf({
-      bip32Path: PropTypes.string,
-      method: PropTypes.string,
-      signature: PropTypes.string,
-    }).isRequired,
-    signingKeys: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+    signatureImporters: PropTypes.shape({}).isRequired,
```

Comment:
This shouldn't be for this release, but we should really have a collection of the various "Types" used in the caravan store that are passed in to these `shape`s that can be imported. This makes sure that: 
a) we have consistent data shapes throughout the app
b) we're actually doing proptype checking here
c) we don't have to repeat code when they are used in multiple places

Reply:
Yeah, I was thinking about that after copy/pasting these a few times so a big :+1: from me.